build.sbt

```diff
@@ -1,7 +1,7 @@
 import com.typesafe.tools.mima.core._
 
 // https://typelevel.org/sbt-typelevel/faq.html#what-is-a-base-version-anyway
-ThisBuild / tlBaseVersion := "0.12" // your current series x.y
+ThisBuild / tlBaseVersion := "0.13" // your current series x.y
```

Comment:
Should have been 0.12 since it was bumped in another PR. I'll fix it on the branch I'm about to merge.

Reply:
Well spotted. thanks